scripts/materials-helper.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+apt-get install -y --no-install-recommends python3 python3-pip
+pip3 install gdown
+
+GDOWN_RETRY_MAX=5
+
+function gdown {
+  local retries=${GDOWN_RETRY_MAX}
+  until [ ${retries} -lt 1 ]; do
+    command gdown $@
+    if [ $? -eq 0 ]; then
+      break
+    fi
+    retries=$(( retries - 1 ))
+    echo "gdown failed, retrying in 1 seconds..." >&2
+    sleep 1
+  done
+  if [ ${retries} -lt 1 ]; then
+    echo "gdown failed 5 times, giving up." >&2
```

Comment:
"5 times" hard coded text could be base on the variables `${GDOWN_RETRY_MAX}`

Reply:
Good catch, I variabilized this afterward.

Reply:
Fixed!